xilem_web/src/one_of.rs

```diff
@@ -137,90 +118,63 @@ where
         };
     }
 
-    fn with_downcast_a(
-        elem: &mut Mut<'_, Self::OneOfElement>,
-        f: impl FnOnce(Mut<'_, Pod<N1, P1>>),
-    ) {
+    fn with_downcast_a(elem: &mut Mut<'_, Self::OneOfElement>, f: impl FnOnce(Mut<'_, Pod<N1>>)) {
```

Comment:
Usually I try to avoid macros, but in that case it might be an idea to reduce boilerplate?

Reply:
When I initially added this, I thought about it, but I don't think it's really worth the added mental complexity, it's only a little bit duplicated boilerplate. When we would have more than 9 variants, the situation may be different.